src/webhooks/index.ts

```diff
@@ -45,6 +45,7 @@ export default function (
           await handleSubscriptionCanceled(
```

Comment:
Perhaps this switch has now grown big enough to have an event bus who handles these events (just a suggestion)

Reply:
I would say a map serialized or an event bus could be useful, but I will not add that complexity increase rn